src/Deposit.js

```diff
@@ -294,66 +313,68 @@ export default class Deposit {
         `keep at address ${keepAddress}...`
     )
     const web3 = factory.config.web3
-    const contract = new web3.eth.Contract(DepositJSON.abi, depositAddress)
-    contract.options.from = web3.eth.defaultAccount
-    contract.options.handleRevert = true
-    const keepContract = new web3.eth.Contract(
-      BondedECDSAKeepJSON.abi,
+    const contract = EthereumHelpers.buildContract(
+      web3,
+      /** @type {TruffleArtifact} */ (DepositJSON).abi,
+      depositAddress
+    )
+    const keepContract = EthereumHelpers.buildContract(
+      web3,
+      /** @type {TruffleArtifact} */ (BondedECDSAKeepJSON).abi,
       keepAddress
     )
-    keepContract.options.from = web3.eth.defaultAccount
-    keepContract.options.handleRevert = true
```

Comment:
Curious why we're dropping setting these options for `contract` and `keepContract?

Reply:
We aren't! I introduced `EthereumHelpers.buildContract` in the previous PR, which replaces our manual `new web3.eth.Contract` invocations and takes care of setting this stuff in one place. This change moves `keepContract` to use that function, so we don't have to do that explicitly here :tada:

Reply:
😎 Very cool. 😅 I do remember seeing that new method in the last PR, but I missed it anyway lol 🙃